clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v1/security/KubernetesV1Credentials.java

```diff
@@ -294,4 +295,32 @@ public Boolean isRegisteredImagePullSecret(String secret, String namespace) {
     }
     return secrets.contains(secret);
   }
+
+  @Override
+  public boolean equals(Object o) {
```

Comment:
Similar to the comment above, `@EqualsAndHashCode` would remove some boilerplate code here.